.travis.yml

```diff
@@ -125,7 +128,9 @@ install:
 script:
  - getconf _NPROCESSORS_ONLN
  - ${MAKE} -j$(getconf _NPROCESSORS_ONLN) check VERBOSE=1 V=${ENV_VERBOSE}
- - ant build test
+ - if [ -e "/usr/lib/jvm/java-8-oracle/" ]; then export JAVA_HOME=/usr/lib/jvm/java-8-oracle/; fi
```

Comment:
Could you move this to the before_install section, before the `java -version` line, and add an `echo "JAVA_HOME=$JAVA_HOME"` after it?